index.js

```diff
@@ -17,3 +17,154 @@ openMenu.addEventListener('click', show);
+                <div class="modal-body">
+                  <p class="mobile-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the
+                    industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and
+                    scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting ever
+                    since the 1500s, when an unknown printer took a galley of type veris lapoa todoe. </p>
+                    <p class="destop-text">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scramble.</p>
+                  </div>
+                <div class="story-languages">
+                  <ul>
+                    ${portfolio.technologies.map((tech) => `<li>${tech}</li><li>
+                      <img class="rectangle" src="images/Rectangle70.png" alt="rectangle" />
+                    </li>`).join('')}
+                  </ul>
+                </div>
+                <div class="popupButton">
+                  <button type="button" class="button button-1 modal-btn">
+                  <a href="${portfolio.live}">
+                    See Live </a>
+                    <img src="images/see-live.svg" alt="see live">
+                  </button>
+                  <button type="button" class="button button-1 modal-btn">
+                  <a href="${portfolio.source}">
+                    See Source </a>
+                    <img src="images/gitHub.svg" alt="see live">
+                  </button>
+                </div>
+              </div>
+            </div>
+            <div id="overlay"></div>
+          </div>
+
+    `).join('');
+
+const openModalButtons = document.querySelectorAll('[data-modal-target]');
+const closeModalButtons = document.querySelectorAll('[data-close-button]');
+const overlay = document.getElementById('overlay');
+
+function openModal(modal) {
+  if (modal == null) return;
+  modal.classList.add('active');
+  overlay.classList.add('active');
+}
+
+function closeModal(modal) {
+  if (modal == null) return;
+  modal.classList.remove('active');
+  overlay.classList.remove('active');
+}
+
+openModalButtons.forEach((button) => {
+  button.addEventListener('click', () => {
+    const modal = document.querySelector(button.dataset.modalTarget);
+    // eslint-disable-next-line no-use-before-define
```

Comment:
- [ ] Kindly avoid disabling linters in your project as this is bad practice, always fix linter errors and ensure all checks are green 👍🏿